src/sage/modular/modsym/all.py

```diff
@@ -1,9 +1,10 @@
+from sage.misc.lazy_import import lazy_import
 
 from .element import set_modsym_print_mode
 
 from .modsym import ModularSymbols, ModularSymbols_clear_cache
 
-from .heilbronn import HeilbronnCremona, HeilbronnMerel
+lazy_import('sage.modular.modsym.heilbronn', ['HeilbronnCremona', 'HeilbronnMerel'])
 
```

Comment:
How do you decide that this should be lazy-imported?

Reply:
It has a compile-time dependency on FLINT and won't be shipped in sagemath-schemes

Reply:
You meant sagemath-modular?